checking.rb

```diff
@@ -0,0 +1,44 @@
+
+module Bank
+  class CheckingAccount < Account
+    attr_reader :checks_used
+
+    def initialize(id, initial_balance, open_date)
+      super(id, initial_balance, open_date)
+      @checks_used = 0
+    end
+
+    def withdraw(withdraw_amount)
+      fee = 100
+      if withdraw_amount > (@balance + 100)
```

Comment:
It seems like you'd want to use the `fee` variable in this calculation rather than a hard-coded `100`